### PR TITLE
Misc utilities

### PR DESCRIPTION
**Pull request by Jaksuhn**

### CODE REVIEW

File: ECommons/GameFunctions/ObjectFunctions.cs
Comment (Limiana) [inline ECommons/GameFunctions/ObjectFunctions.cs:1]: Kinda redundant for just a single comparison since you can just use GetNameplateKind

**Comment by Limiana** [comment]
Thank you! If possible, could you please add some docs on added elements briefly describing what they do and how to use them?

File: ECommons/GameFunctions/ObjectFunctions.cs
Comment (Jaksuhn) [reply ECommons/GameFunctions/ObjectFunctions.cs:1]: Yeah that's fair. The function predated GetNameplateKind and when adding it I guess I didn't realise how redundant that was. Removed

**Comment by Jaksuhn** [comment]
> Thank you! If possible, could you please add some docs on added elements briefly describing what they do and how to use them?

Added doc strings for everything. If any of them are still unclear (probably the excel combos lol) let me know

**Comment by Limiana** [comment]
Thank you!

**Comment by Limiana** [comment]
> Gets a node given a chain of node IDs

I personally don't understand much, is there any example of how it can be used?

**Comment by Jaksuhn** [comment]
> > Gets a node given a chain of node IDs
> 
> I personally don't understand much, is there any example of how it can be used?

I use it semi-frequently when needing to iterate through list sub-nodes.

Example: The pattern for the text node in each tab of the FC chest is `1, 9, 9 + idx, 9`. The text node isn't accessible from root via an ID since it's part of the sub-UldManager, so this method just makes it easier to access in those instances.

**Pull request merged**